platforms/hyperledger-indy/charts/indy-node/templates/statefulset.yaml

```diff
@@ -48,7 +48,11 @@ spec:
 
           curl --request POST --data '{"jwt": "'"$KUBE_TOKEN"'", "role": "ro"}' $VAULT_ADDR/v1/auth/kubernetes/${NODE_ID}/login | jq -j '.auth.client_token' > token;
           X_VAULT_TOKEN=$(cat token);
-          validateVaultResponse 'vault login token' "${X_VAULT_TOKEN}";
+          response=$(curl -s -o /dev/null -w "%{http_code}" $VAULT_ADDR/v1/sys/health)
```

Comment:
you query the vault health check AFTER you query the vault for a token?

Reply:
Ok I need to pay more attention. updated